core/src/main/java/org/locationtech/proj4j/datum/Grid.java

```diff
@@ -376,7 +376,7 @@ private static DataInputStream resolveGridDefinition(String gridName) throws IOE
         // search path for grid definition files, but for now we only check the
         // working directory and the classpath (in that order.)
         File file = new File(gridName);
-        if (file.exists()) return new DataInputStream(new FileInputStream(file));
+        if (file.exists()) return new DataInputStream(new BufferedInputStream(new FileInputStream(file)));
```

Comment:
That's the culrprit, apparently only definitions stored in the resources folder were properly loading.